gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-apis.yaml

```diff
@@ -1079,6 +1079,28 @@ paths:
                                 $ref: "#/components/schemas/Integration"
                 default:
                     $ref: "#/components/responses/Error"
+    /environments/{envId}/integrations/{integrationId}:
+        parameters:
+            - $ref: "#/components/parameters/envIdParam"
+            - $ref: "#/components/parameters/integrationIdParam"
```

Comment:
I may missed it but I think this param est not define 🤔

Reply:
<img width="468" alt="image" src="https://github.com/gravitee-io/gravitee-api-management/assets/155629548/4b0e0d54-65c1-4dd0-8c0a-d0eb4568c66d">

I checked and it was added with previous PR

Reply:
I missed that :sorry: